fairlearn/reductions/_exponentiated_gradient/exponentiated_gradient.py

```diff
@@ -22,6 +22,11 @@ class ExponentiatedGradient(BaseEstimator, MetaEstimatorMixin):
     The exponentiated gradient algorithm is described in detail by
     `Agarwal et al. (2018) <https://arxiv.org/abs/1803.02453>`_.
 
+    .. versionadded:: 0.3.0
```

Comment:
I think this algorithm was there from the start (v0.1) but renamed in v0.3.0. This was just a function when we started out.

That would be a very meaningful "versionchanged" note to add.